File: cash/cash.c
#include<cs50.h>
#include<stdio.h>
#include<unistd.h>
#include<string.h>
#include<math.h>
int main(void)
{
    int coinCount = 0;
    float money = get_float("Change owed: ");
    while (money <= 0)
    {
        money = get_float("Change owed: ");
    }
    int moneyRounded = round(money * 100);

    while (moneyRounded > 0)
    {

        if (moneyRounded - 25 >= 0)
        {
            coinCount += 1;
            moneyRounded -= 25;
        }
        else if (moneyRounded - 10 >= 0)
        {
            coinCount += 1;
            moneyRounded -= 10;
        }
        else if (moneyRounded - 5 >= 0)
        {
            coinCount += 1;
            moneyRounded -= 5;
        }
        else if (moneyRounded - 1 >= 0)
        {
            coinCount += 1;
            moneyRounded -= 1;
        }


    }

    printf("Coins: %i\n", coinCount);
}

File: cash py/cash.py
from cs50 import get_float

coinCount = 0
money = 0
while money <= 0:
    money = get_float("Change owed: ")

moneyRounded = round(money * 100)

while moneyRounded > 0:
    if moneyRounded - 25 >= 0:
        coinCount += 1
        moneyRounded -= 25
    elif moneyRounded - 10 >= 0:
        coinCount += 1
        moneyRounded -= 10
    elif moneyRounded - 5 >= 0:
        coinCount += 1
        moneyRounded -= 5
    elif moneyRounded - 1 >= 0:
        coinCount += 1
        moneyRounded -= 1
print("Coins:", coinCount)